src/cts/src/TritonCTS.tcl

```diff
@@ -88,6 +88,7 @@ sta::define_cmd_args "clock_tree_synthesis" {[-wire_unit unit]
 					     [-apply_ndr] \
                                              [-insertion_delay] \
                                              [-sink_buffer_max_cap_derate] \
+                                             [-dummy_load]
```

Comment:
Add to the docs.

The name is a bit confusing, perhaps -use_dummy_loads ?

Reply:
Shouldn't we just automatically be doing dummy loads if it makes sense? Why does a user need to manually enable the option?

Reply:
@maliberty , changed to -use_dummy_loads.

@QuantamHD, yes, it makes sense to enable this by default when this is ready.   This is only part one.